### ISSUE TITLE
`import` (ES6) breaks parsing: 'Unexpected Token'

### ISSUE THREAD

**Issue by rob4629** [vkarpov15/acquit#22]
(Please) Make Acquit support ES6. I'm unable to parse any of my tests, due to `Error: Line 1: Unexpected token`. I'm using ES6 syntax, with `import` instead of `require`.

**Comment by vkarpov15** [comment]
Fixed and released in v1.1.0 :+1:

**Comment by rob4629** [comment]
Hey, thanks for fixing this so quickly! The _only_ issue I now see is that it fails to parse the [Spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) operator:

```js
it("Returns 400, when sending invalid data in a request", async () => {
   // Set Country_Code to a number
  const newOrgData = { organization: { ...org, country_code: 0 } };
  const { id: adminId, authToken } = await authenticateUser(adminUser);
  // TEST CODE
});
```

```sh
Error: Line 165: Unexpected token ...
```
